package.json

```diff
@@ -41,12 +41,13 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "aframe": "github:mozillareality/aframe#remove-text",
+    "aframe": "github:mozillareality/aframe#hubs/master",
```

Comment:
I think this is due to @mqp 's latest changes -- probably needs to be revertted?

Reply:
`remove-text` already got merged into master afaict